File: Lab 6 (Lecture 7 Practice)/t13.go
package main

import "fmt"

func main() {
  var sum, i int = 0, 1

  for ; i <= 100; i += 1 {
    // if i % 2 == 0 {
    //   continue
    // }
    // sum = sum + i

    // Avoid using continue per requirement.
    if i % 2 != 0 {
      sum = sum + i
    }
  }

  fmt.Printf("i = %d, sum = %d\n", i, sum)
}

File: Lecture 7 Practice/t13.c
#include <stdio.h>

// void main () // It's 2017, my dear
int main (void)
{
  int sum = 0;
  int i = 1;

  for (; i <= 100; i++) {
    // if (i % 2 == 0) {
    //   continue;
    // }
    // sum = sum + i;

    // Avoid using continue per requirement.
    if (i % 2 != 0) {
      sum = sum + i;
    }
  }

  printf ("i = %d, sum = %d\n", i, sum);

  return 0;
}
